lib/blocks.php

```diff
@@ -147,6 +150,45 @@ function gutenberg_reregister_core_block_types() {
 
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
 
+/**
+ * Registers block styles for a core block.
+ *
+ * @param string $block_name The block-name.
+ *
+ * @return void
+ */
+function gutenberg_register_core_block_styles( $block_name ) {
+	if ( ! current_theme_supports( 'split-block-styles' ) ) {
+		return;
+	}
+	$block_name = str_replace( 'core/', '', $block_name );
```

Comment:
There were sometimes differences between the block name and it's folder. The one I know about is:

<img width="748" alt="Screen Shot 2020-12-10 at 08 48 56" src="https://user-images.githubusercontent.com/699132/101737048-9dc70900-3ac4-11eb-9a50-0740b1c0ebd4.png">

Reply:
I did another pass at all blocks, `freeform`/`classic` is the only inconsistency. Added a tweak for that in https://github.com/WordPress/gutenberg/pull/25220/commits/2601d4e10eb7db0855dd062954ad2923f855f694